profiling.md

```diff
@@ -53,62 +51,35 @@ Legion Prof outputs its logs in a compressed binary format using `ZLIB`.
 number of nodes to be profiled. (`N` can be less than the total number
 of nodes---this profiles a subset of the nodes.) Use the 
 `-lg:prof_logfile <logfile>` flag to save the output from each node to
 a separate file. The argument to the `-lg:prof_logfile` flag follows
 the same format as for `-logfile`, except that a `%` (to be replaced
-by the node number) is mandatory. Finally, pass
-the resulting log files to `legion_prof.py`.
-
-{% highlight bash %}
-DEBUG=0 make
-./app -lg:prof <N> -lg:prof_logfile prof_%.gz
-$LG_RT_DIR/../tools/legion_prof.py prof_*.gz
-{% endhighlight %}
-
-(Note: there is now a Rust implementation of Legion Prof, which
-supports a subset of the features, but is 5-15x faster than the Python
-implementation above. See [below](#rust-legion-prof) for details.)
-
-This will generate a subdirectory called `legion_prof` under the
-current directory, including a file named `index.html`. Open this file
-in a browser. Note that, if you are attempting to view this file on
-your local computer, modern browser security settings prevent Legion
-Prof from working properly. Please see the instructions for [Using
-Legion Prof Locally](#using-legion-prof-locally).
-
-A sample of Legion Prof's output is shown below.
+by the node number) is mandatory. Finally,
+the resulting log files need to be post processed with `legion_prof`.
 
-![]({{ "/images/profiling/collapsed_profile.png" | relative_url }})
-
-This profile shows the utilization graph of the memories and processors
-during the run. Legion Prof also can also display  more detailed
-information as described in
-[Interacting with a Profile](#interacting-with-a-profile).
-
-### Rust Legion Prof
+## Legion Prof
 
-There is now a Rust implementation of Legion Prof, in addition to
-Python. It supports a subset of the features of Python, but for the
-features that are supported, the outputs are bitwise identical. See
-below for usage instructions for the Rust profilers.
+Before post processing the log files output from running your
+application with `-lg:prof` the post processer must be installed by
+running the following:
 
 {% highlight bash %}
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-cargo install --locked --path legion/tools/legion_prof_rs
-legion_prof prof_*.gz
+cargo install --locked --features=viewer --features=server --features=archiver --features=client --path legion/tools/legion_prof_rs
 {% endhighlight %}
 
 The first line installs Rust. The second installs a `legion_prof`
-binary into `~/.cargo/bin`. The third runs the profiler. The supported
-flags are substantially similar to the Python implementation and can
-be queried with `--help`.
+binary into `~/.cargo/bin`. The supported flags can be queried with
+`legion_prof --help`.
+The profiler has four different modes it can be installed with which
+support various features.
 
-The following features are supported:
+By default all modes support the following features:
```

Comment:
Let's delete from here to the next section. This is obsolete since we are now at feature parity, and anyway a comparison with the legacy profiler doesn't matter now.

Reply:
Done.